Formula/cargo-nextest.rb

```diff
@@ -49,7 +49,7 @@ def install
       EOS
 
       output = shell_output("cargo nextest run 2>&1")
-      assert_match "Executable unittests src/main.rs", output
+      assert_match "Starting 1 tests across 1 binaries", output
```

Comment:
The output change in the previous release was a bug: https://github.com/nextest-rs/nextest/issues/430